end-to-end-test/remote/specs/core/screenshot.spec.js

```diff
@@ -28,6 +28,8 @@ async function waitForAndCheckPlotsTab() {
 function runResultsTestSuite(prefix, options = {}) {
     it(`${prefix} render the oncoprint`, async function() {
         await waitForOncoprint();
+        await (await getElement('body')).moveTo({ xOffset: 0, yOffset: 0 }); // move mouse out of the way
```

Comment:
@gblaih seems like this is redundant with checkElementWithMouseDisabled, no?  so i mean, i just question whether this would really stabilize it give that we are already trying the same thing and it doesn't seem to work?